target/maven-status/maven-compiler-plugin/compile/default-compile/inputFiles.lst

```diff
@@ -1,17 +1,17 @@
-/Users/sophiaparafina/git/spara/mobyStore/src/main/java/com/docker/mobystore/controller/RestApiController.java
```

Comment:
This file should probably not be in source control, we should probably add `target` to `.gitignore` and `git rm` it